twake/backend/node/src/services/console/web/controller.ts

```diff
@@ -178,13 +176,21 @@ export class ConsoleController {
 
   private async authByPassword(email: string, password: string): Promise<AccessToken> {
     const user = await this.userService.users.getByEmail(email);
-    if (user == null) {
+    if (!user) {
       throw CrudExeption.forbidden("User doesn't exists");
     }
-    const [storedPassword, salt] = await this.userService.users.getHashedPassword({ id: user.id });
-    if (!(await this.passwordEncoder.isPasswordValid(storedPassword, password, salt))) {
-      throw CrudExeption.forbidden("Password doesn't match");
+
+    // allow to login in development mode with any password. This can be used to test without the console provider because the password is not stored locally...
+    if (process.env.NODE_ENV !== "development") {
+      const [storedPassword, salt] = await this.userService.users.getHashedPassword({
+        id: user.id,
+      });
```

Comment:
This is the important part, other changes are just code cleaning

Reply:
Could we add some big warning logs in this function to see them if we have this enabled by mistake in one of our futures installations ?